src/MaskedView.web.tsx

```diff
@@ -0,0 +1,71 @@
+import { type ReactNode, useEffect, useRef, useState } from 'react';
```

Comment:
I prefer `.native.tsx` and `.tsx` instead because `.web` isn't supported in all bundlers and needs additional config.